src/toil/statsAndLogging.py

```diff
@@ -148,6 +148,8 @@ def callback(fileHandle: Union[IO[bytes], IO[str]]) -> None:
             if not isinstance(statsStr, str):
                 statsStr = statsStr.decode()
             stats = json.loads(statsStr, object_hook=Expando)
+            if not stats:
```

Comment:
I'm not sure that an empty stats is disallowed, so just skipping them silently is *probably* fine?